pre-registration/pre-registration-application-service/src/main/resources/application.properties

```diff
@@ -30,7 +30,7 @@ audit.service.contextpath: auditmanager
 audit.service.env: http://kernel-auditmanager-service
 audit.url: ${mosip.base.url}/v1/auditmanager/audits
 auth-token-generator.rest.issuerUrl:${mosip.base.url}/keycloak/auth/realms/preregistration
-auth.server.admin.validate.url:https://dev.mosip.net/v1/authmanager/authorize/admin/validateToken
+auth.server.admin.validate.url:https://qa2.mosip.net/v1/authmanager/authorize/admin/validateToken
```

Comment:
Why this change? This property file should not be used in the deployment. In case of local run it should be pointing to dev.mosip.net. 

Remove this change